src/System.Resources.Reader/ref/System.Resources.Reader.cs

```diff
@@ -5,14 +5,7 @@
 // Changes to this file must follow the http://aka.ms/api-review process.
 // ------------------------------------------------------------------------------
 
+using System.Runtime.CompilerServices;
 
-namespace System.Resources
-{
-    public sealed partial class ResourceReader : System.IDisposable
-    {
-        [System.Security.SecurityCriticalAttribute]
-        public ResourceReader(System.IO.Stream stream) { }
-        public void Dispose() { }
-        public System.Collections.IDictionaryEnumerator GetEnumerator() { return default(System.Collections.IDictionaryEnumerator); }
-    }
-}
+[assembly: TypeForwardedTo(typeof(System.Resources.IResourceReader))]
```

Comment:
>IResourceReader [](start = 51, length = 15)

Why are we type forwarding IResourceReader as well if the only one here was ResourceReader?